src/omega.rs

```diff
@@ -0,0 +1,67 @@
+/// Lambert W function.
```

Comment:
If you want to use `LambertW`, let us use it. I realized that there is also `Error`, which is not a Greek letter. So there is no point in sticking to something that might be misleading. And by analogy, the comment here would be “Lambert W functions.” Meaning there are several.

Reply:
I changed the name to LambertW